docs/en/Configuring-Alluxio-with-OBS.md

```diff
@@ -20,8 +20,12 @@ machines.You can either
 [compile the binaries from Alluxio source code](http://alluxio.org/documentation/master/Building-Alluxio-Master-Branch.html),
 or [download the precompiled binaries directly](http://alluxio.org/documentation/master/Running-Alluxio-Locally.html).
 
+OBS under storage is implemented as an under storage extension, see [this doc](UFSExtensions.html) for how to install the OBS extension,
+the OBS extension's **groupId** is `org.alluxio`, **artifactId** is `alluxio-underfs-obs`, **version** should match the version of the 
```

Comment:
For better user experience, you can provide the concrete command of `alluxio extensions install ...` so that user can simply copay and paste to follow

Reply:
done